shell/initialize/client.js

```diff
@@ -780,6 +819,10 @@ function addHotReload($component, depth) {
   };
 }
 
+// generally, my comment is ??????
+// again, have the feeling that a LOT of this stuff isn't needed, but still lack understanding of what this all does
+// very difficult to follow the code for me
+// final comment: We should clean ALL of this tech debt, while we still have the knowledge in our team to understand most of it...
```

Comment:
strictly speaking everything in shell/initialize is nuxt related and we need to trim out the features we no-longer use or are replacing